### PR TITLE
[FIX JENKINS-43014] save to github from non-master branch

### PR DESCRIPTION
**Pull request by kzantow**
# Description

See [JENKINS-43014](https://issues.jenkins-ci.org/browse/JENKINS-43014).

Skipping tests as this is highly dependent on Github integration.

# Submitter checklist
- [x] Link to JIRA ticket in description, if appropriate.
- [x] Change is code complete and matches issue description
- [x] Appropriate unit or acceptance tests or explanation to why this change has no tests
- [x] Reviewer's manual test instructions provided in PR description. See Reviewer's first task below.
- [x] Ran Acceptance Test Harness against PR changes.

# Reviewer checklist
- [ ] Run the changes and verified the change matches the issue description
- [ ] Reviewed the code
- [ ] Verified that the appropriate tests have been written or valid explanation given

### CODE REVIEW

File: blueocean-github-pipeline/src/main/java/io/jenkins/blueocean/blueocean_github_pipeline/GithubScmSaveFileRequest.java [hunk 1]
```diff
@@ -83,7 +82,7 @@ public Object save(@Nonnull String apiUrl, @Nullable String owner, @Nullable Str
             if(!StringUtils.isBlank(sha)){
                 body.put("sha", sha);
             }
-            final Map ghResp = HttpRequest.put(String.format("%s/repos/%s/%s/contents/%s",
+            final Map<?,?> ghResp = HttpRequest.put(String.format("%s/repos/%s/%s/contents/%s",
```
Comment (michaelneale) [inline blueocean-github-pipeline/src/main/java/io/jenkins/blueocean/blueocean_github_pipeline/GithubScmSaveFileRequest.java]: The java

**Comment by michaelneale** [comment]
running ATH over this: https://ci.blueocean.io/job/ATH-Jenkinsfile/job/master/1898/

File: blueocean-github-pipeline/src/main/java/io/jenkins/blueocean/blueocean_github_pipeline/GithubScmSaveFileRequest.java (hunk 1 above)
Comment (kzantow) [inline blueocean-github-pipeline/src/main/java/io/jenkins/blueocean/blueocean_github_pipeline/GithubScmSaveFileRequest.java]: I don't like to see warnings all the time... probably just an Eclipse thing, maybe I should change my settings...

**Comment by kzantow** [comment]
ATH passing 👍

File: blueocean-github-pipeline/src/main/java/io/jenkins/blueocean/blueocean_github_pipeline/GithubScmSaveFileRequest.java (hunk 1 above)
Comment (michaelneale) [inline blueocean-github-pipeline/src/main/java/io/jenkins/blueocean/blueocean_github_pipeline/GithubScmSaveFileRequest.java]: nah it is ok - just generics letting the team down

**Comment by michaelneale** [comment]
:bee: I checked it as part of https://github.com/jenkinsci/blueocean-pipeline-editor-plugin/pull/35 and it passed the sniff test

GO GO GO

**Pull request merged**